examples/gameserver.yaml

```diff
@@ -93,9 +93,9 @@ spec:
   # counters: # counters are int64 counters that can be incremented and decremented by set amounts. Keys must be declared at GameServer creation time.
   #   games: # arbitrary key.
   #     count: 1 # initial value.
-  #     capacity: 100 # (Optional) Maximum value for the counter. 0 is max(int64).
+  #     capacity: 100 # (Optional) Defaults to 1000 and setting capacity to max(int64) may lead to issues and is not recommended.
```

Comment:
```suggestion
  #     capacity: 100 # (Optional) Defaults to 1000 and setting capacity to max(int64) may lead to issues and is not recommended [#3636](https://github.com/googleforgames/agones/issues/3636)
```

Reply:
We should link back to the GitHub issue so that people can reference the "why". You may need to run `make site-server` to make sure the link functions correctly on the Agones website https://agones.dev/site/docs/reference/gameserver/.